CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Minor: Added chatter count for each category in viewer list. (#3683)
 - Minor: Added option to open a user's chat in a new tab from the usercard profile picture context menu. (#3625)
 - Bugfix: Fixed live notifications for usernames containing uppercase characters. (#3646)
+- Bugfix: Fixed live notifications not getting updated for closed streams going offline. (#3678)
```

Comment:
This Changelog will likely cause a wording conflict in the future, since this fixes fake channels and the Changelog sounds like it applies to normal channels who's tabs you've closed, but meh we'll see what happens.